src/ResourceManager/DataMigration/Commands.DataMigration/help/New-AzureRmDataMigrationSelectedDB.md

```diff
@@ -0,0 +1,175 @@
+---
+external help file: Microsoft.Azure.Commands.DataMigration.dll-Help.xml
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.datamigration/New-AzureRmDataMigrationProject
+schema: 2.0.0
+---
+
+# New-AzureRmDataMigrationSelectedDB
+
+## SYNOPSIS
+Creates a database input object that contains information about source and target databases for migration.
+
+## SYNTAX
+
+### MigrateSqlServerSqlDb (Default)
+```
+New-AzureRmDataMigrationSelectedDB -Name <String> -TargetDatabaseName <String> [-MigrateSqlServerSqlDb]
+ [-MakeSourceDbReadOnly] [-TableMap <System.Collections.Generic.IDictionary`2[System.String,System.String]>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### MigrateSqlServerSqlDbMi
+```
+New-AzureRmDataMigrationSelectedDB -Name <String> -TargetDatabaseName <String> [-MigrateSqlServerSqlDbMi]
+ [-BackupFileShare <FileShare>] [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The New-AzureRmDataMigrationSelectedDB cmdlet creates a database info object that contains information about source and target databases, as well as the table mappings, for migration. This cmdlet can be used as a parameter with the New-AzureRmDataMigrationTask cmdlet.
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> New-AzureRmDataMigrationSelectedDB -Name "HR" -RestoreDatabaseName "HR_PSTEST" -BackupFileShare $backupFileShare
```

Comment:
@hitenjava same comment about providing the corresponding output

Reply:
Also, please add an example for the other parameter set.  This also won't work because it doesn't have either switch parameter.

Reply:
Fixed.